include/openssl/opensslv.h.in

```diff
@@ -86,43 +102,23 @@ extern "C" {
 # define OPENSSL_VERSION_PREREQ(maj,min)                                \
```

Comment:
This is one of "newly added" undocumented symbols (see #9095). While you're doing version related stuff in this PR anyway - please could you document this?

Reply:
No, I don't mind that.